### ISSUE TITLE
[Bug]: where() function on Model is not recognized and thus not autocompleteable

### ISSUE THREAD

**Issue by RoodFruit** [laravel-idea/plugin#512]
### Bug description

The `where()` function does not exist on Models.

```
User::where(     <--- this does not exist in the autocomplete list
```

![1](https://user-images.githubusercontent.com/61903855/154443157-e848a911-05f3-4732-a773-453f1b038be5.png)
![2](https://user-images.githubusercontent.com/61903855/154443160-8bd7b77f-c31b-4ede-b36c-23efb76dc3f2.png)

### Plugin version

5.1.1.213

### Operating system

MacOS

### Steps to reproduce

```
AnyModel::where(     <--- this does not exist in the autocomplete list
```

**Comment by adelf** [comment]
Hi, Rood. 

Have you generated an Eloquent Helper code? main menu > Laravel > Generate Eloquent Helper Code

**Comment by RoodFruit** [comment]
> Hi, Rood.
> 
> Have you generated an Eloquent Helper code? main menu > Laravel > Generate Eloquent Helper Code

Yes, that is why I get the autocomplete list. Normally I need to use `->query()->...` to get the autocomplete list.

**Comment by RoodFruit** [comment]
@adelf when I generate the Eloquent Helper Code, `where()` is NOT part of the `_ide_helper_model_builders.php` file.. hence `where()` is not recognized.

**Comment by adelf** [comment]
It was fixed a long time ago.
